src/tabletop_simulator/importers/card_importer.rb

```diff
@@ -0,0 +1,50 @@
+require "fileutils"
+
+module TabletopSimulator
+  # todo: centralize this
+  module Importers
+    class CardImporter
+      BASE_PATH = %w(Cards).freeze
+      INDIVIDUAL_CARDS_PATH = (BASE_PATH + %w(Individual)).freeze
+
+      def initialize(saved_objects_folder, base_directory)
+        @saved_objects_folder = saved_objects_folder
+        @base_directory = base_directory
+      end
+
+      def import(printed_card)
+        card = imprint(
+          TabletopSimulator::Models::Card.from_template,
```

Comment:
Somewhat interesting choice to be made here- on one hand it feels like the use of the template is more of a concern of the importer rather than the model, on the other I want to keep all the knowledge of the JSON structure of the Card in the model file. 

There might be some solution in the middle, maybe a `Card.from_file(TEMPLATE_FILE)`, then the `Card` class can keep knowledge of how to load a card from a file to itself and while leaving the use and location of the template a concern of the `CardImporter`